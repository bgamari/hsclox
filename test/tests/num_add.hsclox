{
    var a = 1 + 2;
    var b = 4;
    print a+b;
}
