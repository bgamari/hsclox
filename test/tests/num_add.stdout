== code ==
0000    2 OP_CONSTANT         0 '1
0002    | OP_CONSTANT         1 '2
0004    | OP_ADD
0005    3 OP_CONSTANT         2 '4
0007    4 OP_GET_LOCAL     04
0009    | OP_GET_LOCAL     14
0011    | OP_ADD
0012    | OP_PRINT
0013    5 OP_POP
0014    | OP_POP
0015    6 OP_RETURN
7
